draft-ietf-ppm-dap.md

```diff
@@ -1384,20 +1378,26 @@ of !VDAF}}), using the report ID as the nonce:
 ~~~ pseudocode
 (public_share, input_shares) = Vdaf.shard(
     "dap-14" || task_id,
-    measurement, /* plaintext measurement */
-    report_id,   /* nonce */
-    rand,        /* randomness for sharding algorithm */
+    measurement,
+    report_id,
+    rand,
 )
 ~~~
 
-where `task_id` is the task ID. The last input comprises the randomness
-consumed by the sharding algorithm. The sharding randomness is a random byte
-string of length specified by the VDAF. The Client MUST generate this using a
-cryptographically secure random number generator.
+* `task_id` is the task ID.
+
+* `measurement` is the plaintext measurement, represented as the VDAF's
+  `Measurement` associated type.
+
+* `report_id` is the corresponding value from `ReportMetadata`, used as the
+  nonce.
 
-The sharding algorithm will return two input shares. The first input share
-returned from the sharding algorithm is considered to be the Leader's input
-share, and the second input share is considered to be the Helper's input share.
+* `rand` is a random byte string of length specified by the VDAF. Each report's
+  `rand` value MUST be independently sampled from a cryptographically secure
+  random number generator.
+
+The sharding algorithm will return two input shares. The is the Leader's input
+share, and the second is the Helper's.
```

Comment:
Clarity: make discussion of arguments consistent with how we describe structures elsewhere